test/e2e/pages/vl-map.page.js

```diff
@@ -31,12 +31,23 @@ class VlMapPage extends Page {
     return this._getMap('#map-fullscreen');
   }
 
-  async load() {
-    await super.load(Config.baseUrl + '/demo/vl-map.html');
+  async load(url) {
+    await super.load(url || Config.baseUrl + '/demo/vl-map.html');
   }
 
   async _getMap(selector) {
-    return new VlMap(this.driver, selector);
+    const map = await new VlMap(this.driver, selector);
+    await this.driver.wait(async () => {
+      let resolved = false;
+      const promise = this.driver.executeScript('return arguments[0].ready', map);
+      try {
+        await this.driver.wait(async () => await promise.then(() => resolved = true), 1000);
+        return resolved;
+      } catch (error) {
+        return false;
+      }
```

Comment:
Werkt het volgende ook niet?
```suggestion
      try {
        await this.driver.wait(this.driver.executeScript('return arguments[0].ready', map), 1000);
        return true;
      } catch (error) {
        return false;
      }
```

Reply:
heb denk ik elders een gelijkaardig voorstel gedaan

Reply:
@SanderKleykens werkt inderdaad ook, is nog residu toen ik resolved log toegevoegd had om te controleren of de promise resolved wordt in node.